package.json

```diff
@@ -156,8 +156,8 @@
     "postcss-jsx": "^0.36.4",
     "postcss-styled-syntax": "^0.6.4",
     "postcss-syntax": "^0.36.2",
+    "puppeteer": "^22.9.0",
```

Comment:
The order is incorrect now

Reply:
Yes, I noticed that, I'll push a fix for it. I was looking at the E2E tests, and saw a lot of `TypeError: page.waitForTimeout is not a function`.

Reply:
Yeah I had fixed those in #13612 

but the failing taxonomies tests are annoying..maybe we just have to skip those for now